stories/components/Card.complexe.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+import React, { MouseEventHandler, ReactElement } from "react";
+import { ComponentMeta } from "@storybook/react";
+import styled from "styled-components";
+import { Card, Icon } from "../../src";
+import { IconDirection } from "../../src/components/Icon/Direction";
+import { CardStatsDetails } from "../../src/components/Card/CardStats";
+
+export default {
+  title: "Components/Card/Complexe",
+  component: Card,
+} as ComponentMeta<typeof Card>;
+
+interface ExtendedCardProps {
+  title: string;
+  onClick?: MouseEventHandler | null;
+  details: CardStatsDetails[];
+}
+
+export interface ColsProps {
+  cols: number;
+}
+const StyledChevron = styled(Icon.Chevron)`
+  color: ${({ theme }) => theme.colors.primary};
+`;
+
+const StyledWrapper = styled.div<ColsProps>`
+  display: grid;
+  grid-template-columns: ${({ cols }) => `repeat(${cols}, minmax(0, 1fr))`};
+  gap: 1rem;
+`;
+
+const WithLabels = ({ ...props }: ExtendedCardProps): ReactElement => (
```

Comment:
I would destructure here what we need, we are not gaining anything by spreading the props like this

```suggestion
const WithLabels = ({ onClick, title, details }: ExtendedCardProps): ReactElement => (
```

Reply:
? closed but no explanation :(